images/aws-scripts/check-deployment-status.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script is used to check if all deployments are RUNNING
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Allow 5 minutes to wait for kubeflow deployment to be ready
+sleep 5m
+
+ingress_ip=$(kc get ingress istio-ingress -n istio-system  -o json | jq '.status.loadBalancer.ingress' | grep aws)
```

Comment:
Community doesn't have utils with same functionality?  You are check ingress here, but the file title is `check-deployment-status`?